subprojects/wrapper/wrapper.gradle

```diff
@@ -48,7 +48,8 @@ task executableJar(type: Jar) {
 }
 
 integTestTasks.all {
-    inputs.files { project(":distributions").binZip }
+    dependsOn ":distributions:binZip"
+    requiresBinDist = true
```

Comment:
Is there some way we could make it so if you set `requiresBinDist` or `requiresLibsRepo` or anything else that requires that you depend on a particular task to also automatically add the `dependsOn` for you?